src/metadata/ActionMetadata.ts

```diff
@@ -256,7 +256,7 @@ export class ActionMetadata {
     
     static appendBaseRouteToRegexpRoute(route: RegExp, baseRoute: string) {
         if (!baseRoute || baseRoute === "") return route;
-        const fullPath = baseRoute.replace("\/", "\\\\/") + route.toString().substr(1);
+        const fullPath = `^${baseRoute}${route.toString().substr(1)}?$`;
```

Comment:
I don't think so, @pleerock doesn't like dependecies so it won't pass 😜 

If you claim that `.replace("\/", "\\\\/")` was useless it's ok now.
Sample `baseRoute` is `/api` I think, but why `substr(1)` from regex?